MusicApp.Identity.Web/Filters/LoginValidationFilterAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc.Filters;
+using MusicApp.Identity.Application.DTOs;
+using MusicApp.Identity.Application.Validators;
+
+namespace MusicApp.Identity.Web.Filters;
+
+public class LoginValidationFilterAttribute : ActionFilterAttribute
+{
+    private readonly UserLoginDtoValidator _validator;
+
+    public LoginValidationFilterAttribute()
+    {
+        _validator = new();
+    }
+    
+    public override void OnActionExecuting(ActionExecutingContext actionContext)
+    {
+        object playlistObj;
+        actionContext.ActionArguments.TryGetValue("userLoginDto", out playlistObj);
```

Comment:
you don't have to declare playlistObj and cast it to UserRegisterDto outside the TryGetValue method. and it's better to check if playlistObj is not null here